mixin/ang-php@1/example/tests/mixin/AnImpossibleTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Civi\Shimmy\Mixins;
+
+/**
+ * See if mixin lifecycle test cares about exceptions
+ *
+ * @see E2E_Shimmy_LifecycleTest
+ */
+class AnImpossibleTest extends \PHPUnit\Framework\Assert {
+
+  public function testPreConditions($cv): void {
+    $this->assertEquals(1, 1);
+  }
+
+  public function testInstalled($cv): void {
+    $this->assertEquals(1, 1);
+  }
+
+  public function testDisabled($cv): void {
+    $this->assertEquals(1, 1);
+  }
+
+  public function testUninstalled($cv): void {
+    $this->assertEquals(1, 1);
+    // so far so good!
+    if ($cv->isLocal()) {
```

Comment:
I would try outputting what the current errorhandler is and replacing it temporarily, just to see. So just after line 27:

```php
$old = set_error_handler(function($errno, $errstr, $errfile, $errline) { throw new \Exception($errstr); });
fwrite(STDERR, "Old error handler was: " . print_r($old, true));
```

Reply:
Thanks @demeritcowboy - adding output below